src/ontology/uberon-edit.obo

```diff
@@ -222500,6 +222501,12 @@ intersection_of: connects UBERON:0002396 ! vomer
 relationship: dc-contributor https://orcid.org/0000-0001-8415-4848
 relationship: dc-contributor https://orcid.org/0000-0003-2699-3066
 
+[Term]
+id: UBERON:7500112
+name: prosthion
+def: "A craniometric point along a line joining the most oral (anterior) points of the premaxilla. Often, this is midway between the median upper incisor teeth. In ruminants and pigs this point it projected into space." [ISBN:97808736590505]
```

Comment:
Hi @megbalk, I don't have access to the ISBN dbxref, but based on some cursory online searches, this term seems to refer to a **specific point** rather than any point along the line described in the first sentence of the definition. 

I'm not familiar with "oral" being used for anatomical orientation and wonder if "rostral" or just "anterior" would be more appropriate. My searches mostly return results including "most anterior" in their definitions.
"The most anterior point on the maxillary alveolar process in the midline." https://medical-dictionary.thefreedictionary.com/prostheon

Also consider synonyms 'prostheon' and 'alveolar point'.

Reply:
Comma after "Often" should be removed since removing the word "Often" would change the meaning of the sentence.

In the second sentence, I think "it" should be "is".
Comment should have dbxref annotations (I think that is allowed in Uberon).